governance/parameter-docs/param-proportional-kick-incentive.md

```diff
@@ -9,24 +9,24 @@ Technical Docs:
 ```
 
 ## Description
-* What is this parameter?
-* How does it fit in with the protocol and surrounding contracts?
+The Proportional Kick Incentive parameter controls the percentage of the target amount of DAI to raise from bidders (debt + stability fees + liquidation penalty) that will be destined to incentivize keepers when liquidating a vault or resetting an already existing auction.
```

Comment:
"the percentage of the target amount of DAI to raise from bidders (debt + stability fees + liquidation penalty)" - Is there an easier way to say this? It's a bit of a mouthful. Can we simplify to 'Auction Amount' or something?